File: c++/problem-18(RCB)/RCBPLAY.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while(t--){
	    int x,y,z;
	    cin>>x>>y>>z;
	    int need_pt = y-x;
	    int match_pt = 2*z;
	    if (need_pt<=match_pt){
	        cout<<"yes"<<endl;
	    }
	    else{
	        cout<<"no"<<endl;
	    }
	}
	return 0;
}

File: python/problem-19(RCB)/RCBPLAY.py
# cook your dish here
t=int(input())
for i in range(t):
    (x,y,z) = map(int,input().split())
    need_pt=y-x
    match_pt=2*z
    if need_pt<=match_pt:
        print("yes")
    else:
        print("no")